docs/releases.rst

```diff
@@ -1,6 +1,170 @@
 Release notes
 =============
 
+Mailu 2024.03 - 2024-03
+-----------------------
+
+Mailu 2024.03 is available. This release contains new features and many bug fixes. Please at least read the section `upgrading` before attempting to upgrade to the new release.
+
+To make clear you can only go forward with upgrades, we have changed the version naming scheme to Year.Month.Minor.
+It is only possible to downgrade between minor versions (e.g. 2024.03.3 to 2024.03.1).
+
+Highlights
+``````````
+
+Managieve sieve support
+Previously the sieve filters could only be edited via the webmail client. It is now also possible to use an external sieve client for manageming the sieve rules. Configure the sieve client to connect to Mailu on the default sieve port 4190.
+
+RESTful API enhancements
+^^^^^^^^^^^^^^^^^^^^^^^^
+The User interface is enhanced with the quota bytes used (quota_bytes_used) attribute. This attribute states the usage (in bytes) of the mailbox. In combination with the attribute quota_bytes, it is possible to check how much storage an user has left via the RESTful API.
+
+The new `token` endpoint allows the management of authentication tokens. It is recommened to create authentication tokens for all users and configure the email clients to use these authentication tokens for connecting to Mailu.
+
+Force password change
+^^^^^^^^^^^^^^^^^^^^^
+A setting is introduced to force an user to change its password. After changing the password, all sessions are invalidated.
+
+This setting can be configured via:
+* Admin webui
+* Mailu cli command `config-import`
+* RESTful API via the User endpoint and attribute `change_pw_next_login`
+
+Translations
+^^^^^^^^^^^^
+
+The following translations for the Admin webui have been added:
+* Chinese
+* Persion (a.k.a Farsi)
+* Ukrainian
+
```

Comment:
update if #3207 and #3209 make it.

IMHO we should credit the translators here, especially for new translations